src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -168,133 +173,147 @@ const BaseComment = (props: Props) => {
-                            id={createdByUser.id}
-                            name={createdByUser.name}
-                            getUserProfileUrl={getUserProfileUrl}
-                        />
-                    </div>
-                    <div className="bcs-Comment-timestamp">
-                        <ActivityTimestamp date={createdAtTimestamp} />
+                            </TetherComponent>
+                        )}
+                        <div className="bcs-Comment-headline">
+                            <UserLink
+                                data-resin-target={ACTIVITY_TARGETS.PROFILE}
+                                id={createdByUser.id}
+                                name={createdByUser.name}
+                                getUserProfileUrl={getUserProfileUrl}
+                            />
+                        </div>
+                        <div className="bcs-Comment-timestamp">
+                            <ActivityTimestamp date={createdAtTimestamp} />
+                        </div>
+                        <ActivityStatus status={status} />
+                        {isEditing ? (
+                            <CommentForm
+                                isDisabled={isDisabled}
+                                className={classNames('bcs-Comment-editor', {
+                                    'bcs-is-disabled': isDisabled,
+                                })}
+                                updateComment={handleMessageUpdate}
+                                isOpen={isInputOpen}
+                                // $FlowFixMe
+                                user={currentUser}
+                                onCancel={commentFormCancelHandler}
+                                onFocus={commentFormFocusHandler}
+                                isEditing={isEditing}
+                                entityId={id}
+                                tagged_message={tagged_message}
+                                getAvatarUrl={getAvatarUrl}
+                                mentionSelectorContacts={mentionSelectorContacts}
+                                getMentionWithQuery={getMentionWithQuery}
+                            />
+                        ) : (
+                            <ActivityMessage
+                                id={id}
+                                isEdited={isEdited && !isResolved}
+                                tagged_message={tagged_message}
+                                translatedTaggedMessage={translatedTaggedMessage}
+                                {...translations}
+                                translationFailed={error ? true : null}
+                                getUserProfileUrl={getUserProfileUrl}
+                            />
+                        )}
+                    </Media.Body>
+                </Media>
+                {/* $FlowFixMe */}
+                {error ? <ActivityError {...error} /> : null}
+                {!!children && (
```

Comment:
I'm not fully following the component API we're trying to create. It seems likely that we'd want to add additional "child" content to a comment at some point. Should this be rethought and this component decomposed?

Reply:
I think I know what you mean? We could in the future make it a compound component, but I'm not sure if it's necessary for now since we don't know if would even have additional child content to a comment in the future. Replies seems like an obvious `child` of a comment to me, and I think thats why the API in the RFC constructed like this. Maybe @bfoxx1906 has some additional thoughts.